roxctl/connectivity-map/command.go

```diff
@@ -13,7 +12,7 @@ func Command(cliEnvironment environment.Environment) *cobra.Command {
 	c := &cobra.Command{
 		Use:   "connectivity-map <folder-path>",
 		Short: "(Technology Preview) Analyze connectivity based on network policies and other resources.",
```

Comment:
Shouldn't `(Technology Preview)` be removed from the `Short` texts as well?

Reply:
Very good catch!